srv/components/system/storage/enclosure_id/config/sync.sls

```diff
@@ -15,12 +15,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-{% set enclosure = "enclosure-" + ((grains['id']).split('-'))[1] %}
-
-Sync grains data for enclosure_id:
+Sync grains data and mine data for enclosure_id:
```

Comment:
Why do we need separate sync.sls after set.sls?
Also, config.set could be part of config.sls